virttest/qemu_vm.py

```diff
@@ -4300,11 +4312,15 @@ def migrate(self, timeout=virt_vm.BaseVM.MIGRATE_TIMEOUT, protocol="tcp",
             # If we're doing remote migration and it's completed successfully,
             # self points to a dead VM object
             if not not_wait_for_migration:
-                if self.is_alive() and self.is_paused():
+                if self.is_alive():
                     # For short period of time the status can be "inmigrate"
                     # for example when using external program
                     # (qemu commit fe823b6f87b2ebedd692ca480ceb9693439d816e)
-                    self.resume(60)
+                    # resume vm during "inmigrate" status can't work
```

Comment:
The explanation from the commit message is way better. How about something like: `After migration we need to wait for the VM to initialize before sending the "cont". Wait until the status is not "inmigrate".`

(I know that in 5 years I'll be looking at this thinking why are we waiting and why is there inmigrate status...)

Reply:
Oh stupid me... the comment about inmigrate is here...

Reply:
well is this patch actually required? We are already waiting for the "cont" to pass for up to 60s with 0.1s step.

Reply:
@ldoktor  Actually, if we send "cont" in "inmigrate" status, it does not work. Only send "cont" in "paused" status,  vm can be resumed.
This patch will wait the vm status change from "inmigrate" to "paused", then send "cont".
For some other test scenario, vm's status will change from "inmigrate" to "running", in this case, do nothing.

Reply:
> well is this patch actually required? We are already waiting for the "cont" to pass for up to 60s with 0.1s step.
A slight problem, it is wait for 0.1s before the first checking, let me update it with `step=0.1`